templates/practice/documentation/beyond-canonical.html

```diff
@@ -0,0 +1,93 @@
+            <button class="p-side-navigation__toggle--in-drawer js-drawer-toggle" aria-controls="drawer">
+              Toggle table of contents
+            </button>
+          </div>
+          <ul>
+            <li class="p-side-navigation__item">
+              <a class="highlight-link" href="#encounters-and-dialogue">Encounters and dialogue</a>
+            </li>
+            <li class="p-side-navigation__item">
+              <a class="highlight-link" href="#open-source-events">Open-source events</a>
+            </li>
+            <li class="p-side-navigation__item">
+              <a class="highlight-link" href="#supporting-open-source -projects">Supporting open-source projects</a>
+            </li>
+            <li class="p-side-navigation__item">
+              <a class="highlight-link" href="#collaboration-with-practitioners">Collaboration with practitioners</a>
+            </li>
+            <li class="p-side-navigation__item">
+              <a class="highlight-link" href="#scientific-research-community">Scientific research community</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </aside>
+    <main class="col-7">
+      <div id="introduction">
+        <h2>Documentation practice beyond Canonical</h2>
+        <p>We're working to establish patterns of excellence in documentation in Canonical, but not just for Canonical. The scale of operations at Canonical makes it an effective proving-ground for the models and ideas that we're developing &mdash; models and ideas that are being noticed and adopted more widely in the industry.</p>
+      </div>
+      <div class="question-heading" id="encounters-and-dialogue" style="scroll-margin-top: 4.5rem;">
+        <h3>Encounters and dialogue</h3>
+        <p>We actively seek new encounters with problems in documentation, in order that we may understand how our thinking can apply to them. We need to learn from experiences in other contexts.</p>
+        <p>We think, talk and write about the problems and practices of documentation, openly, and with whoever wants to take part part in that dialogue.</p>
+      </div>
+      <div class="question-heading" id="open-source-events" style="scroll-margin-top: 4.5rem;">
+        <h3>Workshops and talks at open-source events</h3>
+        <p>We regularly attend events where the topic of documentation is firmly on the table. Recent examples of participation include <a href="https://2022.pyconuk.org/">PyCon UK 2022</a>, <a href="https://www.youtube.com/watch?v=R4O2WoFC-JQ">DjangoCon Europe 2022</a>, <a href="https://events.canonical.com/event/2/">Ubuntu Summit 2022</a>, <a href="https://www.python.org/events/python-events/1379/">PyCon Namibia 2023</a>.</p>
+        {{ image (
+          url="https://assets.ubuntu.com/v1/a1630e44-working-at-canonical.jpeg",
+          alt="",
+          width="600",
+          height="400",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
```

Comment:
This image has a [`figcaption`](https://vanillaframework.io/docs/patterns/images#image-with-caption) in the copydoc which is missing here.